packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -604,21 +604,24 @@ export class RemoteMediaManager extends EventsScope {
    * Allocates receive slots to all video panes in the current selected layout
    */
   private allocateSlotsToVideoPaneGroups() {
+    // TODO: porentially could be optimized (and the one in which is being called from)
     this.receiveSlotAllocations = {activeSpeaker: {}, receiverSelected: {}};
 
-    this.currentLayout?.activeSpeakerVideoPaneGroups?.forEach((group) => {
-      this.receiveSlotAllocations.activeSpeaker[group.id] = {slots: []};
+    this.currentLayout?.activeSpeakerVideoPaneGroups
+      ?.sort((a, b) => (a.priority > b.priority ? 1 : -1))
```

Comment:
I think we want the opposite sorting as we want the group with the highest priority to be first, right?